lxd/cluster/gateway.go

```diff
@@ -179,9 +179,9 @@ func (g *Gateway) HandlerFuncs(nodeRefreshTask func(*APIHeartbeat)) map[string]h
 
 			nodes := make([]db.RaftNode, 0)
 			for _, node := range heartbeatData.Members {
```

Comment:
Doing so will cause a partially upgraded cluster to completely wipe the raft_nodes table